Documentation/building/facade-code-coverage.bat

```diff
@@ -30,21 +30,36 @@ SET originalfolder=%cd%
 SET sourcefolder=%corefx%\src\%project%\tests
 
 SET coreclrbuild=%coreclr%\bin\Product\Windows_NT.x64.Debug
+SET coreclrbuild=%coreclr%\bin\Product\Windows_NT.x64.Release
```

Comment:
Whoops. Fixed, thanks. Part of my ever on going battle with git to get it to do what I want